src/main/java/net/minecraftforge/server/permission/PermissionAPI.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Minecraft Forge
+ * Copyright (c) 2016.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.server.permission;
+
+import com.google.common.base.Preconditions;
+import com.mojang.authlib.GameProfile;
+import net.minecraftforge.fml.common.FMLLog;
+import net.minecraftforge.server.permission.context.IContext;
+import org.apache.logging.log4j.Level;
+
+import javax.annotation.Nonnull;
+
+public class PermissionAPI
+{
+    private static IPermissionHandler permissionHandler = DefaultPermissionHandler.INSTANCE;
+
+    public static void setPermissionHandler(@Nonnull IPermissionHandler handler)
```

Comment:
Then nobody will be able to write permission mods, since Forge provides one by default. "First come-first served".
Beside that, what's wrong with chaining permission mods? Permission checking is no different than event handling. For troubleshooting, it is enough that Forge logs every handler registration, so that server administrator can see where issues are to be expected.